frontend/src/index.js

```diff
@@ -0,0 +1,55 @@
+let loggedInAccount
+
+document.addEventListener('DOMContentLoaded', () => {
+  console.log('Loaded')
+})
+
+document.getElementById('login-form').addEventListener('submit', (event) => {
```

Comment:
why not have this in a function that is called in the DOMContentLoaded event listener?

Reply:
It doesn't really matter since this get's called on press, rather then page load. As long as we have our script at the bottom of the HTML page. I can move it inside, I just think it's unnecessarily nested to do so

Reply:
To further elaborate, I think putting something inside DOMContent loaded means you want to fire the event right when the content loads, which isn't really true in this case. Therefore I feel the intent is more obvious by leaving it out.